keel/api/v1/notifications/views.py

```diff
@@ -37,30 +38,28 @@ def get_notifications(self, request):
                 description="Failed to get user case",
             )
             response["status"] = 0
-            response["message"] = "Failed to get user case"
-            return Response(response, status=status.HTTP_404_NOT_FOUND)
+            response["message"] = []
```

Comment:
this goes against the current response structure....
Currently all response is recieved in data key

the empty list shoudl also be recieved in the data key
nnot the message

Reply:
fixed